### ISSUE TITLE
React is not defined

### ISSUE THREAD

**Issue by strblr** [aidansunbury/trpc-ui#48]
### Describe the bug
Simple demo code:

```ts
import * as trpcExpress from "@trpc/server/adapters/express";
import { renderTrpcPanel } from "trpc-ui";
import express from "express";
import cors from "cors";
import { appRouter } from "./router";

const app = express();

app.use(cors({ origin: "*" }));

app.use(
  "/trpc",
  trpcExpress.createExpressMiddleware({
    router: appRouter
  })
);

app.get("/panel", (_, res) => {
  res.send(
    renderTrpcPanel(appRouter, {
      url: "http://localhost:3030/trpc",
      meta: {
        title: "tRPC Panel",
        description: "Custom description"
      }
    })
  );
});

app.listen(3030, () => {
  console.log("Server is running on port 3030");
});
```

### To Reproduce
1. Run the code above
2. Go to http://localhost:3030/panel
3. Blank screen + `panel:3572 ReferenceError: React is not defined` in the console

**trpc-ui version:** 1.0.8

**trpc [adapter](https://trpc.io/docs/server/adapters) used:** express

**Are you using superjson:** no

### Expected behavior
Having a working UI.

### Screenshots
The screen is blank due to React being undefined.

**Comment by dandanthedev** [comment]
Having the same problem with a Next.js app.

**Comment by dandanthedev** [comment]
![image](https://github.com/user-attachments/assets/c39d1d2a-1dde-4315-8d31-972a3bc0262d)

**Comment by aidansunbury** [comment]
That's bad on my part, biome thinks react only needs to be imported as a type, but it gives that error if it's not:

<img width="1094" alt="Screenshot 2024-12-27 at 4 40 00 PM" src="https://github.com/user-attachments/assets/fa82fc17-f8e4-4148-88a6-9bb6b6f82773" />

https://github.com/aidansunbury/trpc-ui/commit/1bc703889d2337ce90faac981d0ee9bfc442ccff#diff-918f6c8ad76cb8c4f5a03e80dd11c1a3f266c4ec7df7c9248495900fc6029d78

I'll get it fixed and make sure it doesn't accidentally get reintroduced in a sec. I also am going to try and get preview deployments set up for the test app that uses the preview package.

**Comment by aidansunbury** [comment]
@dandanthedev @strblr Should be fixed in 1.0.10. https://www.npmjs.com/package/trpc-ui. Reopen the issue if it is not.

**Comment by dandanthedev** [comment]
> @dandanthedev @strblr Should be fixed in 1.0.10. https://www.npmjs.com/package/trpc-ui. Reopen the issue if it is not.

Yep, working now!
